### PR TITLE
Increasing environment variables for app container

### PR DESCRIPTION
**Pull request by andreaswittig**
Currently, the number of environment variables is limited to 3. We are limited by the maximum CloudFormation parameters. Nevertheless, I'd like to increase the number of environment variables for the app container to 6. The ambassador and sidecar container typically need less environment variables in my opinion, Therefore, I would not add additional environment variables for those containers.

### CODE REVIEW

**Comment by andreaswittig** [comment]
Test ...

**Comment by andreaswittig** [comment]
test

File: README.md [hunk 1]
```diff
@@ -40,6 +40,12 @@ Resources:
         AmbassadorEnvironment2Value: '' # optional
         AmbassadorEnvironment3Key: '' # optional
         AmbassadorEnvironment3Value: '' # optional
+        AmbassadorEnvironment4Key: '' # optional
```
Comment (michaelwittig) [inline README.md]: Instead of `Ambassador` it should be `App`?

File: README.md [hunk 2]
```diff
@@ -193,6 +199,48 @@ Resources:
       <td>no</td>
       <td></td>
     </tr>
+    <tr>
+      <td>AmbassadorEnvironment4Key</td>
```
Comment (michaelwittig) [inline README.md]: Instead of `Ambassador` it should be `App`?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cfn-modules/fargate-service",
-  "version": "1.1.1",
+  "version": "1.2.1",
```
Comment (michaelwittig) [inline package.json]: should be `1.2.0`. Please also update the CloudFormation output value to `1.2.0`

**Review by michaelwittig — CHANGES_REQUESTED** [review]

**Pull request merged**